src/pods/canvas/components/relation/database-relation-two-tables-path.component.tsx

```diff
@@ -17,24 +17,62 @@ export const DatabaseRelationshipTwoTablePathComponent: React.FC<
   DatabaseRelationshipMiddleProps
 > = ({
   id,
-  relationType,
   startCoords,
+  relationType,
   endCoords,
   isSelected,
   onClick,
   onDoubleClick,
 }) => {
-  console.log('DRAW PATH CONNECTOR');
+  const oneToOneRelationSelfPathLeft = `
+  M ${startCoords.x - TABLE_CONST.TABLE_WIDTH} ${startCoords.y} 
+  H ${startCoords.x - (TABLE_CONST.TABLE_WIDTH + TABLE_CONST.HORIZONTAL_LEFT_EXTENSION)} 
+  V ${endCoords.y} 
+  H ${endCoords.x}
+  `;
+
+  const oneToOneRelationSelfPathRight = `
+  M ${startCoords.x + TABLE_CONST.TABLE_WIDTH} ${startCoords.y}
+  H ${startCoords.x + TABLE_CONST.TABLE_WIDTH + TABLE_CONST.HORIZONTAL_LEFT_EXTENSION}
+  V ${endCoords.y}
+  H ${endCoords.x}
+  `;
+
+  const getRelationPath = () => {
+    if (relationType === '1:1') {
+      return startCoords.x < endCoords.x
+        ? oneToOneRelationSelfPathRight
+        : oneToOneRelationSelfPathLeft;
+    }
+  };
```

Comment:
All this can be extracted to business method and add Unit tests, something like:

And rename to generateRelationPath

NAme make no sense
oneToOneRelationSelfPathRight
oneToOneRelationSelfPathLeft;

Self? We are in two table scenario

Please review, I would expect a business method with something like:

```ts

generateRelationPath = (params needed) => {
  // implementation
}
```

And "vaciar el cangrejo" ;) form the component